jsonrpc/server.go

```diff
@@ -255,7 +264,8 @@ func (s *Server) HandleReadWriter(ctx context.Context, rw io.ReadWriter) error {
 		activated: activated,
 	}
 	msgCtx := context.WithValue(ctx, ConnKey{}, conn)
-	resp, err := s.HandleReader(msgCtx, rw)
+	// header is unnecessary for read-writer(websocket)
+	resp, _, err := s.HandleReader(msgCtx, rw)
```

Comment:
The returned header from s.HandleReader is never used, so I don't think it should be returned